hetmech/test_diffusion.py

```diff
@@ -56,7 +56,7 @@ def test_diffusion_step_row_and_column_damping(
         input_matrix = self.get_clean_matrix()
 
         # Create the matrix expected for simultaneous dual normalization
-        pr, pc = row_damping, column_damping
+        pc, pr = row_damping, column_damping
```

Comment:
I made these tests pass, by this ugly hack... would it be hard to rewrite `expect` so we can get rid of the transpose below and this change?